website/ember-cli-build.js

```diff
@@ -7,43 +7,50 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-module.exports = function (defaults) {
```

Comment:
made this change to allow having conditional logic for the `fingerprint.exclude` setting.